bokeh/core/_templates/autoload_js.js

```diff
@@ -103,9 +105,16 @@ calls it with the rendered model.
   ];
 
   function run_inline_js() {
-    for (var i = 0; i < inline_js.length; i++) {
-      inline_js[i](window.Bokeh);
-    }
+    if (window.Bokeh !== undefined || force === "1") {
+      for (var i = 0; i < inline_js.length; i++) {
+        inline_js[i](window.Bokeh);
+      }
+    } else if (Date.now() < (window._bokeh_init_load+10000)) {
+      setTimeout(run_inline_js, 500);
```

Comment:
What about using a setInterval and clear the interval when window.Bokeh is defined?
I am OK with this implementation but setInterval feels to me like best suited for this use case...